assets/app/views/settings.html

```diff
@@ -72,24 +72,48 @@
       <h2>
         <span>Resource limits</span>
         <i style="margin-left: 5px; cursor: help;" class="small pficon pficon-help" data-toggle="tooltip" data-placement="right"
-          title="Resource limits define minimum and maximum values that can be created for each resource type."></i>
+          title="Resource limits define minimum and maximum values that can be requested for each resource type."></i>
       </h2>
       <div ng-if="(limitRanges | hashSize) == 0"><em>{{emptyMessageLimitRanges}}</em></div>
       <table ng-if="(limitRanges | hashSize) > 0" class="table">
         <thead>
           <th>Resource type</th>
-          <th>Min</th>
-          <th>Max</th>
-          <th>Default</th>
+          <th>
+            Min
+            <i class="small pficon pficon-help" data-toggle="tooltip" data-placement="right"
+          title="The minimum amount of this resource that can be requested."></i>
+          </th>
+          <th>
+            Max
+            <i class="small pficon pficon-help" data-toggle="tooltip" data-placement="right"
+          title="The maximum amount of this resource that can be requested.  The limit must also be below the maximum."></i>
+          </th>
+          <th>
+            Default Request
+            <i class="small pficon pficon-help" data-toggle="tooltip" data-placement="right"
+          title="The request is the amount of the resource that will be applied against your quota, you will be guaranteed to have that much of the resource available.  The default request will be applied to all newly created objects unless another request is specified."></i>
+          </th>
+          <th>
+            Default Limit
+            <i class="small pficon pficon-help" data-toggle="tooltip" data-placement="right"
+          title="The limit defines what the resource will be allowed to burst to.  The default limit will be applied to all newly created objects unless another limit is specified."></i>
```

Comment:
The limit defines the maximum amount of compute resource the container may have access to during execution.  The default limit will be applied if no limit is specified by the container.  If no request is made for the compute resource on the container or via a Default Request value, the container will default to request the specified limit.